iotcore/src/main/java/com/google/android/things/iotcore/IotCoreClient.java

```diff
@@ -21,38 +21,41 @@
 import android.support.annotation.VisibleForTesting;
 import android.util.Log;
 
+import org.eclipse.paho.client.mqttv3.IMqttDeliveryToken;
+import org.eclipse.paho.client.mqttv3.MqttCallback;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttConnectOptions;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
+import org.jose4j.lang.JoseException;
+
 import java.io.EOFException;
 import java.net.SocketTimeoutException;
 import java.net.UnknownHostException;
 import java.security.KeyPair;
 import java.time.Duration;
 import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
 import java.util.Queue;
 import java.util.concurrent.Executor;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Semaphore;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 
-import org.eclipse.paho.client.mqttv3.IMqttDeliveryToken;
-import org.eclipse.paho.client.mqttv3.MqttCallback;
-import org.eclipse.paho.client.mqttv3.MqttClient;
-import org.eclipse.paho.client.mqttv3.MqttConnectOptions;
-import org.eclipse.paho.client.mqttv3.MqttException;
-import org.eclipse.paho.client.mqttv3.MqttMessage;
-import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
-import org.jose4j.lang.JoseException;
-
 import javax.net.ssl.SSLException;
 
 /**
  * IotCoreClient manages interactions with Google Cloud IoT Core for a single device.
  *
  * <p>This class provides mechanisms for using Cloud IoT Core's main features. Namely
  * <ul>
- *     <li>Publishing device telemetry</li>
- *     <li>Publishing device state</li>
- *     <li>Receiving configuration changes</li>
+ * <li>Publishing device telemetry</li>
```

Comment:
nit: no preference, but I suggest to use indent for li.

Reply:
:+1: